frontend/packages/schema-editor/src/components/SchemaInspector/ItemFieldsTab/ItemFieldsTable/ItemFieldType/ItemFieldType.module.css

```diff
@@ -0,0 +1,8 @@
+.linkButton {
+  border: none;
+  background-color: inherit;
+  font-size: inherit;
+  font-family: inherit;
+  padding: 0;
+  text-decoration: none;
```

Comment:
Vi bør beholde understreken for å signalisere at elementet er klikkbart.
```suggestion
```

Reply:
Fixed